SPECS/fluent-bit/fluent-bit.spec

```diff
@@ -83,6 +81,10 @@ Development files for %{name}
 %{_libdir}/fluent-bit/*.so
 
 %changelog
+* Thu Oct 03 2024 Sudipta Pandit <sudpandit@microsoft.com> - 3.0.6-1
```

Comment:
@realsdx can you ensure all of these CVEs are resolved first.  Take them from the 3.0 branch's history

- CVE-2024-25431
- CVE-2024-34250
- CVE-2024-25629
- CVE-2024-28182

Reply:
Sure